env/default/default.properties

```diff
@@ -10,31 +10,31 @@ gauge_reports_dir = reports
 overwrite_reports = true
 
 # Set to false to disable screenshots on failure in reports.
-screenshot_on_failure = true
+screenshot_on_failure = false
 
 # The path to the gauge logs directory. Should be either relative to the project directory or an absolute path
 logs_directory = logs
 
 # Timeout in milliseconds for requests from language
 # runner when connecting to IDE.
-ide_request_timeout = 1200000
+ide_request_timeout = 3600000
```

Comment:
Why did you change these timeouts?

Reply:
Sometimes I see due to n/w issues tests fails due to time out so I kept a bigger value, I would change something to reasonable value